contentful/entry.py

```diff
@@ -80,7 +80,8 @@ def _resolve_include(self, resource, localized, includes):
             localized,
             resource,
             includes,
-            depth=self._depth + 1
+            depth=self._depth + 1,
```

Comment:
This is to step forward the depth, this method call creates the next resource in the include tree, therefore it's depth is current depth + 1